### ISSUE TITLE
Overallocation warning on project becoming confirmed

### ISSUE THREAD

**Issue by rowleya** [UoMResearchIT/CapX#388]
I was looking at CapX capacity and noticed that there are some projects where an "impossible" provisional / unconfirmed assignment has been made.  I am assuming that this is because there was a point where there was an overallocation of unconfirmed projects to someone, then one became confirmed.  I am now wondering if the project managers of the projects that are still provisional are now aware that these assignments would now take the RSE over their maximum allocation and would therefore not work?  

It could therefore be useful to notify the PMs when something like this happens.  Note this could just be a notification that an unconformed project that overlaps with one of your unconfirmed assignments has been confirmed to avoid a more complex check (but this might already be complex enough!).

**Comment by aharwood2** [comment]
Yes this could be useful. I guess there is a two-fold thing here a) warning someone when allocating a resource that by doing so, the resource becomes overallocated at some point during the assignment due to clashing confirmed projects; b) a notification mechanism to email PMs of an unconfirmed or provisional assignment when changes to confirmation status push a resource overallocated due to confirmed projects.

Not easy I don't think but possible and also useful.
